chart/values.yaml

```
@@ -383,6 +393,19 @@ workers:
       maxSurge: "100%"
       maxUnavailable: "50%"
 
+  # When not set, the values defined on podSecurity.securityContext will be used
+  securityContext:
+    runAsUser: 50000
+    fsGroup: 0
```

```suggestion
  securityContext: {}
  #  runAsUser: 50000
  #  fsGroup: 0
```

We shouldn't set these for all of the components, otherwise `podSecurity.securityContext` will never actually be used and isn't an easy way to set it across the board.